mne/source_estimate.py

```diff
@@ -464,15 +464,18 @@ def __init__(self, data, vertices=None, tmin=None, tstep=None,
         self._kernel = kernel
         self._sens_data = sens_data
         self._kernel_removed = False
-        self.times = None
-        self._update_times()
         self.subject = _check_subject(None, subject, False)
 
     @property
     def sfreq(self):
         """Sample rate of the data."""
         return 1. / self.tstep
 
+    @property
+    def times(self):
+        """Timestamps for each sample."""
+        return self.tmin + (self.tstep * np.arange(self.shape[-1]))
```

Comment:
I don't like when properties do non-atomic computations. It can make your code slow because you don't know it does more than accessing a property. That's why we had the _update_times function. I would not change this.